ivy_tests/test_ivy/test_frontends/test_torch/test_pointwise_ops.py

```diff
@@ -1676,6 +1676,39 @@ def test_torch_pow(
     )
 
 
+@handle_frontend_test(
+    fn_tree="torch.float_power",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
+        num_arrays=2,
+        min_value=0,
+        max_value=6,
+        small_abs_safety_factor=10,
+        safety_factor_scale="linear",
+    ),
+)
+def test_torch_float_power(
+    dtype_and_x,
+    on_device,
+    fn_tree,
+    frontend,
+    test_flags,
+):
+    input_dtype, x = dtype_and_x
+    # bfloat16 uses different semantics for exponent and mantissa than other float types
+    assume("bfloat16" not in input_dtype)
```

Comment:
It's best practice to define the unsupported dtypes using a decorator to the function instead of filtering them in the test. Please have a look at [this](https://lets-unify.ai/docs/ivy/overview/deep_dive/data_types.html#supported-and-unsupported-data-types) section of our docs.

Reply:
@AnnaTz, all frameworks support float_power operation on bfloat16 tensors, the issue is that bfloat16 uses only 7 bits for the mantissa part, while other float types (float16, float32, float64) use at least10 mantissa bits:

![bfloat16](https://storage.googleapis.com/gweb-cloudblog-publish/images/Three_floating-point_formats.max-700x700.png)

Some of the test cases will fail when comparing a bfloat16 tensor (returned from Tensorflow/JAX) with a float64 tensor (returned from torch/numpy). One of them is:
```text
inp = tensor(2, dtype=bfloat16)
exp = tensor(1.25, dtype=bfloat16)
out = float_power(inp, exp)
```
Tensorflow/JAX will return a bfloat16 of 2.375, while torch/numpy will return a float64 tensor of 2.3784.
To avoid potential failures, I have marked out all bfloat16 cases as bad samples.

Reply:
The torch documentation states "If neither input is complex returns a torch.float64 tensor, and if one or more inputs is complex returns a torch.complex128 tensor.". So, I think you can just cast the result of the frontend function to the correct dtype.